frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx

```diff
@@ -122,29 +123,97 @@ export const SimpleSessionRecordingsFilters = ({
                     allowNew={false}
                     openOnInsert
                 />
-                <LemonMenu
-                    items={[
-                        items.length > 0 && {
-                            title: 'Preferred properties',
-                            items: items,
-                        },
-                        {
-                            items: [
-                                {
-                                    label: `${
-                                        displayNameProperties.length === 0 ? 'Add' : 'Edit'
-                                    } person display properties`,
-                                    to: urls.settings('project-product-analytics', 'person-display-name'),
-                                },
-                            ],
-                        },
-                    ]}
+                <Popover
+                    visible={showPropertySelector}
+                    onClickOutside={() => setShowPropertySelector(false)}
+                    overlay={
+                        <Configuration
+                            properties={quickFilterProperties}
+                            onChange={(value) => {
+                                setQuickFilterProperties(value)
+                            }}
+                        />
+                    }
+                    referenceElement={buttonRef.current}
+                    placement="right-start"
                 >
-                    <LemonButton size="small" type="secondary" icon={<IconPlus />}>
-                        Choose quick filter
-                    </LemonButton>
-                </LemonMenu>
+                    <LemonMenu
+                        items={[
+                            defaultItems.length > 0 && { items: defaultItems },
+                            personPropertyItems.length > 0 && {
+                                title: 'Person properties',
+                                items: personPropertyItems,
+                            },
+                        ]}
```

Comment:
then we could label the items in the menu with the person or event icon so people can see what they have a little?